modules/stroeerCoreBidAdapter.js

```diff
@@ -0,0 +1,194 @@
+import {registerBidder} from '../src/adapters/bidderFactory.js';
+import {BANNER} from '../src/mediaTypes.js';
+import * as utils from '../src/utils.js';
+
+const BIDDER_CODE = 'stroeerCore';
+const DEFAULT_HOST = 'hb.adscale.de';
+const DEFAULT_PATH = '/dsh';
+const DEFAULT_PORT = '';
+const FIVE_MINUTES_IN_SECONDS = 300;
+const USER_SYNC_IFRAME_URL = 'https://js.adscale.de/pbsync.html';
+
+const isSecureWindow = () => utils.getWindowSelf().location.protocol === 'https:';
+const isMainPageAccessible = () => getMostAccessibleTopWindow() === utils.getWindowTop();
+
+function getTopWindowReferrer() {
+  try {
+    return utils.getWindowTop().document.referrer;
+  } catch (e) {
+    return utils.getWindowSelf().referrer;
+  }
+}
+
+function getMostAccessibleTopWindow() {
+  let res = utils.getWindowSelf();
+
+  try {
+    while (utils.getWindowTop().top !== res && res.parent.location.href.length) {
+      res = res.parent;
+    }
+  } catch (ignore) {
+  }
+
+  return res;
+}
+
+function elementInView(elementId) {
+  const resolveElement = (elId) => {
+    const win = utils.getWindowSelf();
+
+    return win.document.getElementById(elId);
+  };
+
+  const visibleInWindow = (el, win) => {
+    const rect = el.getBoundingClientRect();
+    const inView = (rect.top + rect.height >= 0) && (rect.top <= win.innerHeight);
+
+    if (win !== win.parent) {
+      return inView && visibleInWindow(win.frameElement, win.parent);
+    }
+
+    return inView;
+  };
+
+  try {
+    return visibleInWindow(resolveElement(elementId), utils.getWindowSelf());
+  } catch (e) {
+    // old browser, element not found, cross-origin etc.
+  }
+  return undefined;
+}
+
+function buildUrl({host: hostname = DEFAULT_HOST, port = DEFAULT_PORT, securePort, path: pathname = DEFAULT_PATH}) {
+  if (securePort) {
+    port = securePort;
+  }
+
+  return utils.buildUrl({protocol: 'https', hostname, port, pathname});
+}
+
+function getGdprParams(gdprConsent) {
+  if (gdprConsent) {
+    const consentString = encodeURIComponent(gdprConsent.consentString || '')
+    const isGdpr = gdprConsent.gdprApplies ? 1 : 0;
+
+    return `?gdpr=${isGdpr}&gdpr_consent=${consentString}`
+  } else {
+    return '';
+  }
+}
+
+export const spec = {
```

Comment:
hey @Kubitini , I see that in docs PR you wrote that bidder gvlid is 136, why didn't you include in the bidder spec, so gdpr enforcement module could pick it up?
You can look for examples in other bid adapters:

```
gvlid: 136,
```

Reply:
Hi @aleksatr thank you for the notice, I've added it there.